src/main/java/entities/DesignableMaze.java

```diff
@@ -0,0 +1,5 @@
+package entities;
+
+public class DesignableMaze extends Maze{
+
```

Comment:
add attributes: 
- addBlock(location): which adds a wall to the maze's state at the specified location 
- removeBlock(location): which removes a wall to the maze's state at the specified location

where location is a 2 element array of integers e.g.: [row, column]